src/testnet/helium/node.rs

```diff
@@ -64,14 +71,59 @@ pub struct Node {
     burnchain_tip: Option<BlockSnapshot>,
     pub chain_state: StacksChainState,
     chain_tip: Option<StacksHeaderInfo>,
-    config: Config,
+    pub config: Config,
     keychain: Keychain,
     last_sortitioned_block: Option<SortitionedBlock>,
     mem_pool: MemPoolFS,
     event_dispatcher: EventDispatcher,
     nonce: u64,
 }
 
+lazy_static! {
+    static ref DEFAULT_CONNECTION_OPTIONS: ConnectionOptions = ConnectionOptions {
+        inbox_maxlen: 100,
+        outbox_maxlen: 100,
+        timeout: 5000,
+        idle_timeout: 15,               // how long a HTTP connection can be idle before it's closed
+        heartbeat: 60000,
+        // can't use u64::max, because sqlite stores as i64.
+        private_key_lifetime: 9223372036854775807,
+        num_neighbors: 4,
+        num_clients: 1000,
+        soft_num_neighbors: 4,
+        soft_num_clients: 1000,
+        max_neighbors_per_host: 10,
+        max_clients_per_host: 1000,
+        soft_max_neighbors_per_host: 10,
+        soft_max_neighbors_per_org: 100,
+        soft_max_clients_per_host: 1000,
+        walk_interval: 9223372036854775807,
+        dns_timeout: 15_000,
+        max_inflight_blocks: 6,
+        .. std::default::Default::default()
```

Comment:
Should we move this to `config.rs` in a new `p2p` struct, so that we can make this settings override-able by the `toml` config?